libraries/chain/include/eosio/chain/protocol_feature_manager.hpp

```diff
@@ -12,23 +12,25 @@ enum class protocol_feature_t : uint32_t {
 };
 
```

Comment:
Add comment to both `builtin_protocol_feature_t` and `protocol_feature_t` that these values are included in the protocol digest so there values must remain constant.